website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
 
 # Docker
 
-Docker is an open-source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into containers—standardized executable components combining application source code with the operating system (OS) libraries and dependencies required to run that code in any environment.
+Docker is an open-source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into standardized executable components called containers. These containers combine the app's source code with the operating system (OS) libraries and dependencies required to run that code in any environment.
 
 <object data="https://www.youtube.com/embed/Tde7GqE6FQQ?autoplay=0" width='750px' height='400px'></object> 
 
 ## Prerequisites
 
 * [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
-* [Docker-Compose ](https://docs.docker.com/compose/install/)(version 1.29.2 or later)
+* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later) _(Recommended)_
 
-Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+1. Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+2. `cd` into the installation folder.
 
-`cd` into the installation folder.
-
-## Quick Start (with docker-compose)
+:::tip
+If you'd like to name your installation directory something other than `appsmith`, update your docker-compose.yml file with the name you'd like:
+```yml
+# docker-compose.yml
 
-:::info
-Appsmith [auto-update](./#enabling-appsmith-auto-updates-with-docker-compose) now also supports auto-backup feature. This backup can be used to roll back an update to a previous version of Appsmith, if the user wants to.
+# ...
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: <YOUR_DIRECTORY_NAME>
+# ...
+```
 :::
 
-The Appsmith Docker image is built with all the components required to run within a single Docker container. All these multiple processes are managed by a Supervisord instance, which is a lightweight process manager.
-
-:::note
-You can expect one container running when using the `docker-compose ps` command. Something like:
+## Setup with docker-compose
```

Comment:
Isn't it just Docker Compose and not docker-compose?

Reply:
Good point, I've been using `docker-compose` successfully but according to [this StackOverflow question](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose) it looks like the original `docker-compose` plugin is being deprecated and its features are being integrated into the newer Docker Desktop, which uses the `docker compose` syntax.

@sum35h or @sharat87 , do you know if it's safe to update all references to `docker-compose` and replace them with `docker compose`? Just want to make sure the behavior should be the same since I'm not super experienced with Docker.